docs/Project-Configuration/Stepback-Bisection.md

```diff
@@ -0,0 +1,51 @@
+# Stepback Bisection
```

Comment:
I'd add that the project flag added as part of this [ticket](https://jira.mongodb.org/browse/DEVPROD-977) should also be documented on [this page](https://docs.devprod.prod.corp.mongodb.com/evergreen/Project-Configuration/Project-and-Distro-Settings#project-flags) (and it should be linked to from this page)

Reply:
The project configuration file can actually not have stepback bisection so I did not update it (it's only configurable via the project settings page. I could include a mention but I think doc day + project settings/configuration simplifying project will cover it. I think it's good to just forgo mentioning on that page since it isn't supported.